java/src/jmri/jmrix/loconet/bdl16/BDL16Panel.java

```diff
@@ -420,32 +499,39 @@ private void getComboBox(int n, JPanel jp) {
      * @param n2 second OpSw number
      * @param jp the JPanel into which the JComboBox is placed
      */
-    private void getComboBox(int n, int n2, JPanel jp) {
+    private JComboBox getComboBox(int n, int n2) {
         String number = Integer.toString(n);
         if (number.length() == 1) {
-            number = "0"+number;
+            number = "0" + number;
         }
         String number2 = Integer.toString(n2);
         if (number2.length() == 1) {
-            number2 = "0"+number2;
+            number2 = "0" + number2;
         }
-        
-        jp.add(new JLabel("OpSw"+number+" and OpSw"+number2+": ")); // NOI18N
+
         comboBox[n] = new JComboBox<>(
-                new String[] {Bundle.getMessage("COMBOBOX_TEXT_OPSW"+number+"_THROWN_OPSW"+number2+"_THROWN"), 
-                        Bundle.getMessage("COMBOBOX_TEXT_OPSW"+number+"_CLOSED_OPSW"+number2+"_THROWN"),
-                        Bundle.getMessage("COMBOBOX_TEXT_OPSW"+number+"_THROWN_OPSW"+number2+"_CLOSED"),
-                        Bundle.getMessage("COMBOBOX_TEXT_OPSW"+number+"_CLOSED_OPSW"+number2+"_CLOSED")
+                new String[] {Bundle.getMessage("COMBOBOX_TEXT_OPSW" + number + "_THROWN_OPSW" + number2 + "_THROWN"),
+                        Bundle.getMessage("COMBOBOX_TEXT_OPSW" + number + "_CLOSED_OPSW" + number2 + "_THROWN"),
+                        Bundle.getMessage("COMBOBOX_TEXT_OPSW" + number + "_THROWN_OPSW" + number2 + "_CLOSED"),
+                        Bundle.getMessage("COMBOBOX_TEXT_OPSW" + number + "_CLOSED_OPSW" + number2 + "_CLOSED")
                 });
-        jp.add(comboBox[n]);
+        // default choice = 0 so already set
+        // size all combos to match the widest one (and a little bit more)
+        comboBox[n].setPreferredSize(new JLabel("XXXXXXX" + Bundle.getMessage("COMBOBOX_TEXT_OPSW36_THROWN")).getPreferredSize());
```

Comment:
Tricky combination of bundle and non-bundle strings.  May need `// NOI18N` annotation.